src/routes/rules/components/RulesNav.vue

```diff
@@ -112,4 +115,4 @@ watch(() => props.activeTitle,
     font-size: 2.5rem !important;
   }
 }
-</style>
+</style>
```

Comment:
Keep the whitespace at EOF
```suggestion
</style>

```